src/containers/costume-tab.jsx

```diff
@@ -82,31 +130,24 @@ class CostumeTab extends React.Component {
             return null;
         }
 
-        const addBackdropMsg = (
-            <FormattedMessage
-                defaultMessage="Add Backdrop"
-                description="Button to add a backdrop in the editor tab"
-                id="gui.costumeTab.addBackdrop"
-            />
-        );
-        const addCostumeMsg = (
-            <FormattedMessage
-                defaultMessage="Add Costume"
-                description="Button to add a costume in the editor tab"
-                id="gui.costumeTab.addCostume"
-            />
-        );
-
-        const addMessage = target.isStage ? addBackdropMsg : addCostumeMsg;
-        const addFunc = target.isStage ? onNewBackdropClick : onNewCostumeClick;
+        const addLibraryMessage = target.isStage ? messages.addLibraryBackdropMsg : messages.addLibraryCostumeMsg;
+        const addBlankMessage = target.isStage ? messages.addBlankBackdropMsg : messages.addBlankCostumeMsg;
+        const addLibraryFunc = target.isStage ? onNewLibraryBackdropClick : onNewLibraryCostumeClick;
 
         return (
             <AssetPanel
-                buttons={[{
-                    message: addMessage,
-                    img: addCostumeIcon,
-                    onClick: addFunc
-                }]}
+                buttons={[
+                    {
+                        message: addBlankMessage,
+                        img: addBlankCostumeIcon,
+                        onClick: this.handleNewBlankCostume
+                    },
+                    {
+                        message: addLibraryMessage,
+                        img: addLibraryCostumeIcon,
```

Comment:
Should we switch this from the cat to the add backdrop icon when you're on the stage? (../components/stage-selector/icon--backdrop.svg)

Reply:
Ah I didn't realize that was there! Will do